content/docs/studio/create-view.md

```diff
@@ -57,11 +71,20 @@ You will need to specify advanced settings in the following scenarios.
   Enter the full file path, and specify whether the file is for `Metrics` or
   `Parameters`.
 
-## Share a view
+## DVC Studio GitHub app installation
 
-You can share your views on the web. Simply click on the button labelled
-`Private` next to the name of the view. In the menu that pops up, turn on
-`Share to Web`. ![](https://static.iterative.ai/img/studio/view_share.png)
+If you are using a GitHub account, you will have to install the DVC Studio
+GitHub app on your GitHub repositories / organizations that you want to use with
+DVC Studio. When you try to create a view, if the app is not already installed,
+DVC Studio will not be able to fetch the list of your GitHub repositories. In
+this case, you will be prompted to configure Git integrations as shown below.
 
-You can change a shared (public) view and make it private by turning off
-`Share to web`.
+![](https://static.iterative.ai/img/studio/configure_git_integrations.png)
+
+Click on the link to `Configure Git integrations settings`. This will open the
+`Git integrations` section of your profile page.
+
+![](https://static.iterative.ai/img/studio/configure_github.png)
+
+Click on the `Configure` button, and you will be redirected to the GitHub page
+to install and authorize the DVC Studio GitHub app.
```

Comment:
If there is time: it might also be good to document the steps that users can take on GitHub + add screenshots.

It might also be worthwhile to note that users can only give us access to the repositories they want to use with Studio. They don't have to grant us access to all repositories in their GitHub org if they don't want to.

Reply:
Thanks @duijf I've added a note regarding limiting the access to only the required repositories. 
Regarding documenting the steps to be taken on GH, I was thinking that since that part is external to us, maybe we should not add details/screenshots for that, coz we don't have control on if/when there's any change in that. What do you think?

Reply:
Added notes below for limiting access.

Reply:
Cool! Thanks :)

Regarding screenshots: I feel like users might benefit from the additional docs more than what this would cost us in terms of maintenance.

While the UI on GitHub's side may change slightly, the main concepts should continue to apply. (If the flow changes in major ways, they will probably announce this somehow). Having the screenshots be slightly outdated won't be the end of the world I think + it's relatively quick to fix after we find out.

Your call though!

Reply:
Got it. I'll create a follow-up ticket to add more GH instructions/screenshots, coz I think it needs some restructuring - will be good to move this section into a new page, coz the current one is quite long already and has too many screenshots.

Reply:
now it's a comment here https://github.com/iterative/dvc.org/issues/2515